contracts/SportMarkets/Parlay/ParlayMarketsAMM.sol

```diff
@@ -347,6 +391,35 @@ contract ParlayMarketsAMM is Initializable, ProxyOwned, ProxyPausable, ProxyReen
     }
 
     function exerciseParlay(address _parlayMarket) external nonReentrant notPaused onlyKnownMarkets(_parlayMarket) {
+        _exerciseParlay(_parlayMarket);
+    }
+
+    function exerciseParlayWithOfframp(
+        address _parlayMarket,
+        address collateral,
+        bool toEth
+    ) external nonReentrant notPaused onlyKnownMarkets(_parlayMarket) {
+        ParlayMarket parlayMarket = ParlayMarket(_parlayMarket);
+        address parlayOwner = parlayMarket.parlayOwner();
+        require(msg.sender == parlayOwner, "Only allowed from parlay owner");
```

Comment:
Maybe is okey to be triggered by anyone, just needs the `parlayOwner` allowance to be checked for the further conversion. 

On the other hand, for security reasons it is okey to have the owner mandatory trigger the exercise. 
The telegram bot integrators may prefer the version that anyone can trigger the exercise. It may allow them freedom to add fees and etc.

Reply:
I dont think it makes sense to allow someone else to decide how I want to claim (sUSD vs other collaterals), at least not in its current form.